asistente_ladm_col/gui/supplies/wiz_supplies_etl.py

```diff
@@ -281,6 +285,22 @@ def show_message(self, message, level, duration=10):
         self.bar.clearWidgets()  # Remove previous messages before showing a new one
         self.bar.pushMessage(message, level, duration)
 
+    def show_resumen_etl(self):
+        self.ladm_tables_feature_count_after = {layer.name(): layer.featureCount() for layer in self.ladm_layers}
+        summary = """<html><head/><body><p>"""
+        summary += """<hr>"""
+        summary += QCoreApplication.translate("SuppliesETLWizard", "New report on the migration of cadastral data to the input model of the cadastral manager:<br/>")
+        for layer in self.ladm_tables_feature_count_before:
+            summary += QCoreApplication.translate(
+                        "SuppliesETLWizard",
+                        '<br/><b>{}</b> : {}'.format(
+                            layer, self.ladm_tables_feature_count_after['{}'.format(layer)] - 
+                            self.ladm_tables_feature_count_before['{}'.format(layer)]))
```

Comment:
Given that `layer` is a layer name, `'{}'.format(layer)` isn't just the same as `layer`? :open_mouth: 
Try to double check what you commit, to make the code easier to read. :pray: